resources/views/admin/user/edit.blade.php

```diff
@@ -0,0 +1,62 @@
+<?php ?>
+@extends('admin.layout.index')
+
+@section('content')
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">User</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row -->
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        Edit user
+                    </div>
+                    <div class="panel-body">
+                        <div class="row">
+                            <div class="col-lg-6">
+                                <form role="form">
+                                    <div class="form-group">
+                                        <label>Username</label>
+                                        <input class="form-control">
+                                    </div>
+                                    <div class="form-group">
+                                        <label>Email</label>
+                                        <input class="form-control" type="email">
+                                    </div>
+                                    <div class="form-group">
+                                        <label>Password</label>
+                                        <input class="form-control" type="password">
+                                    </div>
+                                    <div class="form-group">
+                                        <label>Only see blog</label>
```

Comment:
Since it is your start for the usage input of 'Only see' and 'Can delete' , please provide alter sql code for current working table in develop

Reply:
Fixed, i add columns can_delete, can_see in table user default nullable